src/pybind/mgr/dashboard_v2/frontend/src/app/core/auth/login/login.component.ts

```diff
@@ -0,0 +1,37 @@
+import { Component, OnInit, ViewContainerRef } from '@angular/core';
+import { Login } from './login.model';
+import { AuthService } from '../../../shared/services/auth.service';
+import { Router } from '@angular/router';
+import { HttpClient } from '@angular/common/http';
+import { ToastsManager } from 'ng2-toastr';
+
+@Component({
+  selector: 'oa-login',
```

Comment:
s/oa-login/dashboard-login/ maybe?

Reply:
Angular components prefix renamed from 'oa' to 'cd'